rfc/rfc-0010-drop-core-runtime.md

```diff
@@ -0,0 +1,37 @@
+# RFC-#: Drop Core Runtime
+
+**Authors:** @scothis
+
+**Status:**
+
+**Pull Request URL:** [#1373](https://github.com/projectriff/riff/pull/1373)
+
+**Superseded by:** N/A
+
+**Supersedes:** N/A
+
+**Related:** N/A
+
+## Problem
```

Comment:
Do we want to mention the extra cost of maintaining, testing, and supporting the core runtime in addition to the Knative runtime?

Reply:
added